src/main/resources/assets/minestuck/models/block/flowery_mossy_stone_bricks.json

```diff
@@ -1,6 +0,0 @@
-{
-    "parent": "block/cube_all",
-    "textures": {
-        "all": "minestuck:block/flowery_mossy_stone_bricks"
-    }
-}
```

Comment:
When replacing this with the numbered models, you forgot to update the model used for the double-slab block in the `flowery_mossy_stone_brick_slab` blockstate file (which still refers to the `flowery_mossy_stone_bricks` model)
Since you changed the slab models to use the same textures as the `flowery_mossy_stone_bricks1`model, it makes sense to fix this by changing the model in the blockstate to `flowery_mossy_stone_bricks1`.